<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>690</width>
      <height>420</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>监控画面</string>
    </property>
    <widget class="QWidget" name="ImgWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>670</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
   </widget>

    <widget class="QPushButton" name="cameraButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>获取摄像头</string>
    </property>
   </widget>
 
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>获取视频文件</string>
    </property>
   </widget>

   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   
  
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>